src/components/policy/policy_external/include/policy/policy_helper.h

```diff
@@ -178,6 +178,14 @@ struct CheckAppPolicy {
    */
   void AddResult(const std::string& app_id, PermissionsCheckResult result);
   /**
+   * @brief Adds result of check of applications related to current application
+   * policy with updated one
+   * @param app_id Application id
+   * @param result Result value
+   */
+  void AddRelatedAppsResult(const std::string& app_id,
+                            PermissionsCheckResult result);
```

Comment:
@AKalinich-Luxoft is it output parameter? So rename it

Reply:
@LuxoftAKutsan param was renamed as well as whole function name. See https://github.com/smartdevicelink/sdl_core/pull/1779/commits/c40c209107a48effdaf494913f9e62abd9d92334